common/production_methods/babylon_government.txt

```diff
@@ -241,31 +224,91 @@ pm_general_studies = {
 		}
 	}
 }
+
 pm_no_initiatives = {
 	texture = "gfx/interface/icons/production_method_icons/ownership_aristocrats.dds"
 }
-pm_urban_academia = {
+
+pm_bookshops = {
 	texture = "gfx/interface/icons/production_method_icons/government_run.dds"
+	unlocking_technologies = {
+		dialectics
+	}
 	building_modifiers = {
-		
 		level_scaled = {
 			building_employment_academics_add = 25
-            building_employment_clerks_add = 50
-            building_employment_laborers_add = 75
+			building_employment_clerks_add = 50
+			building_employment_shopkeepers_add = 75
 		}
 		workforce_scaled = {
-			goods_input_paper_add = 0.20
-			goods_input_services_add = 0.20
-		}	
+			goods_input_paper_add = 1
+			goods_input_services_add = 1
+		}
 	}
 	state_modifiers = {
-		level_scaled = {
```

Comment:
and level scaled because it is supposed to be a tool for large uneducated nations to get workers in a state. So it can't be depended on having those workers first.